consensus/hotstuff/packer/packer.go

```diff
@@ -0,0 +1,54 @@
+package packer
```

Comment:
I had to extract this logic into separate package to allow using it from other places. There is a hard circular dependency if we try to use `consensus/hotstuff/signature` in `state/protocol/bader/seed`. Extracting just RLP encoding solves the problem.